mobile/src/main.jsx

```diff
@@ -22,6 +23,8 @@ import Settings from './containers/Settings'
 import { loadState, saveState } from './lib/localStorage'
 import { init } from './lib/cozy-helper'
 
+Raven.config('https://29bd1255b6d544a1b65435a634c9ff67@sentry.cozycloud.cc/2').install()
```

Comment:
Cool! I suggest we add in the `ProvidePlugin` (https://github.com/cozy/cozy-files-v3/blob/master/config/webpack.target.mobile.js#L19-L22) a compil variable named `__SENTRY_TOKEN__` and we inject it like:

```js
Raven.config(`https://${__SENTRY_TOKEN__}@sentry.cozycloud.cc/2`).install()
```

it'll be injected properly on build.